chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/dialog/README.md

# @vaadin/dialog

> A web component used to present information and user interface elements in an overlay.

Reviewer: ```suggestion
A web component for presenting information and user interface elements in an overlay.
```

Author: Done.
